src/components/ConversationSummary.vue

```diff
@@ -0,0 +1,190 @@
+<template>
+  <div class="blip-container summary">
+    <div :class="'bubble ' + position">
+      <div v-if="document.isLoading">
+        <div class="typing">
+          <div class="typing-dot"></div>
+          <div class="typing-dot"></div>
+          <div class="typing-dot"></div>
+        </div>
+      </div>
```

Comment:
I think we have the same loading behavior on component `ChatState`. Do you think it makes sense to create a component for this loading and use it in both components?